runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/getPotentialCapabilities/gpc001.c

```diff
@@ -151,7 +154,7 @@ Java_com_ibm_jvmti_tests_getPotentialCapabilities_gpc001_verifyOnLoadCapabilitie
     	error(env, JVMTI_ERROR_INTERNAL, "Unexpected number [%d] of unavailable capabilities. Expected 0", unavailableCount);
     	return JNI_FALSE;
     }
-
+#if JAVA_SPEC_VERSION >= 9
     if (JVMTI_VERSION_9_0 == env->jvmtiVersion) {
```

Comment:
I expect this should be `JVMTI_VERSION_9_0 <= env->jvmtiVersion` (here and in similar conditions below).